packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -3304,7 +3304,7 @@ describe('network stubbing', { retries: 15 }, function () {
           done()
         })
 
-        cy.intercept('/should-err*', function (req) {
+        cy.intercept('/should-err*', { times: 1 }, function (req) {
```

Comment:
This change was to fix the flake. I would like to understand this problem better and fix the root cause hopefully in a follow up PR.

Reply:
can you add a `TODO` or `NOTE` to this test indicating the change and the expected followup?